### DISCUSSION TITLE
Does Umami Support multiple domain names?

### DISCUSSION THREAD

**Discussion by mkusold** [umami-software/umami#844] [Q&A]
If I have two or more domain names for a single site eg. (a.com and b.com that both point to one site), does Umami support having one unified dashboard?

**Comment by mikecao** [comment]
All that matters is the website id. It doesn't matter what the actual domain is.

**Reply by clementmas** [reply]
The data can be recorded across multiple domains but can it be grouped/displayed by domains on the dashboard or the API?

**Reply by ramazansancar** [reply]
> The data can be recorded across multiple domains but can it be grouped/displayed by domains on the dashboard or the API?

No, as far as I know. Whatever the path is from each domain, that's what's displayed on the UI screen. I don't think it differentiates based on domain.
